nncf/tensorflow/quantization/init_range.py

```diff
@@ -65,15 +65,15 @@ def get_init_config_for_quantization_point(self, layer: tf.keras.layers.Layer, i
     def get_init_config_for_scope_and_group(self, node_name: str, group: QuantizerGroup) -> RangeInitConfig:
         matches: List[RangeInitConfig] = []
         for pl_config in self.per_layer_range_init_configs:
-            if should_consider_scope(
+            should_be_considered = should_consider_scope(
                 node_name, ignored_scopes=pl_config.ignored_scopes, target_scopes=pl_config.target_scopes
-            ):
-                if group == pl_config.target_group or pl_config.target_group is None:
-                    matches.append(
-                        RangeInitConfig(
-                            pl_config.init_type, pl_config.num_init_samples, pl_config.init_type_specific_params
-                        )
+            )
+            if should_be_considered and (group == pl_config.target_group or pl_config.target_group is None):
+                matches.append(
+                    RangeInitConfig(
+                        pl_config.init_type, pl_config.num_init_samples, pl_config.init_type_specific_params
                     )
+                )
```

Comment:
Was this also an automatic refactor? I'd suppose creating `should_be_considered ` is unnecessary.

Reply:
No, but without it autoformatting looks bad 
```python
            if should_consider_scope(qid, pl_config.ignored_scopes, pl_config.target_scopes) and (
                group == pl_config.target_group or pl_config.target_group is None
            ):
```
With `should_be_considered` variable more clear.

Reply:
Ok, I see. I'd personally would choose the autoformatted version just to be unified with the rest of the code, because otherwise it's unclear when to create a separate variable for a condition and when not to create. But this way is also fine if you like it more.